src/Promenade.Web/Views/Shared/_Layout.cshtml

```diff
@@ -182,10 +207,19 @@
                                 if (item.ChangeToLinkWhenExtraSmall || item.Navigations.Count() == 0)
                                 {
                                     <li role="presentation" class="nav-item prom-nav-link">
-                                        <a aria-label="@item.NavigationText.AriaLabel"
-                                           href="@item.Link"
-                                           class="nav-link prom-nav-link"
-                                           title="@item.NavigationText.Title">@item.NavigationText.Label</a>
+                                        @if (string.IsNullOrEmpty(item.NavigationText.Label)
+                                            && !string.IsNullOrEmpty(item.Icon)
+                                            && !string.IsNullOrEmpty(item.NavigationText.Title))
+                                        {
+                                            <a href="@item.Link"
+                                               class="nav-link prom-nav-link"
+                                               title="@item.NavigationText.Title">@item.NavigationText.Label</a>
```

Comment:
The label text here will always be empty, it should probably be swapped for the icon.

Reply:
Resolved in commit 9b9fd46116bc0d3399fe1289b3f3852d4fd5576c